app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -62,11 +62,13 @@ function DocumentCaptureReviewIssues({
         altFailedDocTypeMsg={isFailedDocType ? t('doc_auth.errors.doc.wrong_id_type_html') : null}
         hasDismissed={hasDismissed}
       />
-      <p>
-        <HtmlTextWithStrongNoWrap
-          text={t('idv.failure.attempts_html', { count: remainingSubmitAttempts })}
-        />
-      </p>
+      {remainingSubmitAttempts !== Infinity && (
```

Comment:
do you also want to do a `> 0` check on the number of attempts?

Reply:
it should be rate limited before remainingSubmitAttempts < 0 which'd be pretty concerning ... if that scenario was to arise, it could be helpful to allow it to happen to surface that deeper problem? 🤔